src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -85,7 +85,6 @@ public void parseCommand_help() throws Exception {
     @Test
     public void parseCommand_list() throws Exception {
         assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD) instanceof ListCommand);
-        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
```

Comment:
If this expected behavior is false, would it be better to add another assertFalse statement?